arrays/practiceObjectOriented/main_test.go

```diff
@@ -0,0 +1,11 @@
+package main
+
+import "testing"
+
+func TestMonolithic(t *testing.T) {
+	area, peri := Monolithic(10, 5)
```

Comment:
good to see that you are writing test cases 👍

Reply:
thank you